mifospay/src/main/java/org/mifos/mobilewallet/mifospay/payments/ui/SendFragment.java

```diff
@@ -213,10 +213,11 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 return;
             }
             double amount = Double.parseDouble(etAmount.getText().toString());
-            MakeTransferFragment fragment = MakeTransferFragment.newInstance(externalId,
-                    amount);
-            fragment.show(getChildFragmentManager(),
-                    Constants.MAKE_TRANSFER_FRAGMENT);
+            if (!mTransferPresenter.checkSelfTransfer(externalId)) {
+                mTransferPresenter.checkBalanceAvailability(externalId, amount);
+            } else {
+                showSnackbar(Constants.SELF_ACCOUNT_ERROR);
```

Comment:
The strings which belong to UI must be in `strings.xml` file. Otherwise, in multiple languages support, we can not translate those strings which are in java file and it will lead to the bad user experience.

Reply:
I am not creating new strings and only using strings defined earlier as I did not want edit other code where these strings are used. Maybe we can create another PR for replacing already defined strings.